src/cml.js

```diff
@@ -508,7 +508,7 @@ class CML {
     if (fetchDepth !== undefined) {
       if (fetchDepth <= 0) {
         if ((await exec('git rev-parse --is-shallow-repository')) === 'true') {
-          return await exec('git fetch --all --unshallow');
+          return await exec('git fetch --all --tags --unshallow');
         }
       } else {
         return await exec(`git fetch --all --depth=${fetchDepth}`);
```

Comment:
probably this too? What are the pros/cons?

```suggestion
        return await exec(`git fetch --all --tags --depth=${fetchDepth}`);
```

Reply:
Would this fetch tags for commits you don't have? feels like that could cause issues.

If it only pulls tags for your fetch-depth range, then sure.

Reply:
seems fine from some quick testing.